spine-lua/AtlasAttachmentLoader.lua

```diff
@@ -70,7 +70,7 @@ function AtlasAttachmentLoader:newMeshAttachment (skin, name, path)
 end
 
 function AtlasAttachmentLoader:newSkinningMeshAttachment (skin, name, path)
-	return SkinningMeshAttachment.new(name)
+	return nil --SkinningMeshAttachment.new(name)
```

Comment:
I couldn't find information on this `nil --` construct. Could you tell me what it does?

Reply:
SkinningMeshAttachment is an undefined global. So I would get an error on SkinningMeshAttachment.new(),  so I just made it nil. I only have Spine's essential version, so I don't have a good way to test the mesh stuff.

Are we missing a "spine-lua.attachments.SkinningMeshAttachment" library?